guillotina/traversal.py

```diff
@@ -447,9 +447,14 @@ async def real_resolve(self, request: IRequest) -> Optional[MatchInfo]:
                     )
 
         if not view and len(tail) > 0:
-            # we should have a view in this case because we are matching routes
-            await notify(TraversalViewMissEvent(request, tail))
-            raise HTTPNotFound(content={"reason": "object and/or route not found"})
+            # Try arbitrary "path" in the path
+            try:
+                view_name = tail[0] + "?"
+                view = query_multi_adapter((resource, request), method, name=view_name)
+            except AttributeError:
```

Comment:
I don't think this should ever raise `AttributeError`

If `view` is None here, shouldn't you be doing the `notify` and raise HTTPNotFound?